.github/workflows/ci.yml

```
@@ -0,0 +1,171 @@
+
+      - name: Approve native builds
+        run: pnpm approve-builds
+
+      - name: Run unit tests
+        run: pnpm test
+
+      - name: Run integration tests
+        run: pnpm test:integration
+
+      - name: Upload coverage
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)